locale/es/LC_MESSAGES/messages.po

```diff
@@ -6412,7 +6412,7 @@ msgstr "Detalles del tema"
 msgid "Supply a pretty URL for your detail page."
 msgstr "Proporciona una URL amigable para tu página de detalles."
 
-# 86%
+# 86%
```

Comment:
Those are line ending changes (consistent over all files now), before it was a weird mix of different styles which I unified